ydb/core/tx/schemeshard/schemeshard_export_scheme_uploader.cpp

```diff
@@ -0,0 +1,261 @@
+#include "schemeshard.h"
+#include "schemeshard_export_scheme_uploader.h"
+#include "schemeshard_impl.h"
+#include "schemeshard_path_describer.h"
+
+#include <ydb/core/tx/schemeshard/schemeshard_export_helpers.h>
+#include <ydb/core/wrappers/abstract.h>
+#include <ydb/core/wrappers/s3_storage_config.h>
+#include <ydb/core/wrappers/s3_wrapper.h>
+#include <ydb/library/actors/core/actor_bootstrapped.h>
+#include <ydb/library/actors/core/hfunc.h>
+
+namespace NKikimr {
+namespace NSchemeShard {
```

Comment:
В новом коде можно писать короче:
```suggestion
namespace NKikimr::NSchemeShard {
```